libminifi/src/c2/C2Client.cpp

```diff
@@ -177,7 +177,7 @@ std::shared_ptr<state::response::ResponseNode> C2Client::loadC2ResponseConfigura
             std::static_pointer_cast<state::response::ObjectNode>(prev_node)->add_node(new_node);
           }
         } else {
-          std::string optionName = option + "." + name;
+          std::string optionName = std::string(option).append(".").append(name);
```

Comment:
should we go with `StringUtils::join_pack` for such string concatenations?

Reply:
Good idea, updated in 3b7a34d4fbb31fbb01acfa3654723ea05239f0fa